src/utils/api.js

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import config from './config';
+
+const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJpbWlyZSIsImVtYWlsIjoiaW1pcmVAZ21haWwuY29tIiwiaWF0IjoxNTQxNTgyMjQzLCJleHAiOjE1NDE1ODk0NDN9.KqR2wO0Q94VjC3zr7CqwYYz-_6HrQY0j63jiBUILlcw';
```

Comment:
Store this token in the environment variables

Reply:
Done